lib/pdf.js

```diff
@@ -85,7 +85,7 @@ PDF.prototype.toFile = function PdfToFile (filename, callback) {
 
 PDF.prototype.exec = function PdfExec (callback) {
   var callbacked = false
-  var child = childprocess.spawn(this.options.phantomPath, [].concat(this.options.phantomArgs, [this.script]))
+  var child = childprocess.spawn(this.options.phantomPath, [].concat(this.options.phantomArgs, [this.script]), this.options.cliOptions)
```

Comment:
NP. Other than `detached`, I didn't find the options really useful, in this case. But I also didn't want to hardwire the `detached` option in. IMO, exposing these won't hurt anybody.